css/components/alerts.css

```diff
@@ -0,0 +1,28 @@
+
+.alert{
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 0.5rem 1rem;
+  background-color:  var(--palesilver-li);
+  width:30rem;
+  border-radius: 0.5rem;
+  color: var(--prim-col);
+  box-shadow: 1px 1px 3px 1px lightgrey;
+  box-sizing: border-box;
```

Comment:
colors can be used in variables , if you change in future it will be easy to change